packages/blade/src/components/Input/_decisions/decisions.md

```diff
@@ -0,0 +1,212 @@
+| errorText                                                                                          | `string`                                                                  | No                                                                                  |                                                           | Shown when the `validationState` of the input field is set to Error. Only one of `helpText`, `errorText` or `successText` is shown at a time in the priority order as `errorText`, `successText`, `helpText`                                                                          |
+| successText                                                                                        | `string`                                                                  | No                                                                                  |                                                           | Shown when the `validationState` of the input field is set to Success. Only one of `helpText`, `errorText` or `successText` is shown at a time in the priority order as `errorText`, `successText`, `helpText`                                                                        |
+| autoFocus                                                                                          | `boolean`                                                                 | No                                                                                  | `false`                                                   | The autofocus global attribute is a Boolean attribute indicating that an element should be focused on page load. [Web Reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus), [Native Reference](https://reactnative.dev/docs/textinput#autofocus) |
+| fullWidth                                                                                          | `boolean`                                                                 | No                                                                                  | `false`                                                   | **Discuss if we want to add support for full-width input fields.**                                                                                                                                                                                                                    |
```

Comment:
When we say `fullWidth` what do we mean? Are we going to provide a large fixed width or a `min-width` to the input field (assuming the normal input field would be have a `width: 100%` and would take the width of its container).

Reply:
yes it was pending on design, we will remove this prop and all our input fields will be width:100% to the container.